changelog

```diff
@@ -1,3 +1,10 @@
+turnkey-dokuwiki-17.0 (1) turnkey; urgency=low
+
+  * Note: Please refer to turnkey-core's 17.0 changelog for changes common to all
+    appliances. Here we only describe changes specific to this appliance.
+
+ -- Zhenya Hvorostian <zhenya@turnkeylinux.org>  Wed, 06 Feb 2022 17:05:38 +0300
```

Comment:
This should include version. Or at least note it's the current latest version, something like

`* Updated to latest stable` or, if you can figure out the actual release info, something like:
`* Updated to latest stable: 2020-07-29 "Hogfather"` (previous changelog entry)

Not sure how, though probably requires building it and inspecting the files.

Reply:
I'll take care of this @OnGle . It's sort of cheating, but I'll try building it and (assumnig it all goes well) I'll update the changelog after the  fact...